### PR TITLE
fix: [sc-2886] Config errors in musig and ghost

### PR DESCRIPTION
**Pull request by teghnet**
Solution to using multiple chains at once

Related to:
- https://github.com/chronicleprotocol/musig/pull/15

Needed for:
- https://github.com/chronicleprotocol/app-of-apps/pull/15

### CODE REVIEW

File: config-spectre.hcl [hunk 1]
```diff
@@ -34,8 +35,43 @@ spectre {
     }
   }
 
+  dynamic "scribe" {
+    for_each = {
+      for p in length(var.spectre_pairs) == 0 ? var.data_symbols : var.spectre_pairs : p=>
+      var.scribe_contracts[var.spectre_target_network][p]
+      if contains(try(keys(var.scribe_contracts[var.spectre_target_network]), []), p)
```
Comment (deep-quality-dev) [inline config-spectre.hcl:42]: The correct name of `scribe_contracts` should be `optimistic_scribe_contracts` and `scribe` is using `scribe_contracts` I think.

File: config-spectre.hcl [hunk 2]
```diff
@@ -34,8 +35,43 @@ spectre {
...
+      feeds = env("CFG_FEEDS", "") == "*" ? concat(var.feed_sets["prod"], var.feed_sets["stage"]) : try(var.feed_sets[env("CFG_FEEDS", "prod")], explode(",", env("CFG_FEEDS", "")))
+
+      # Name of the pair to fetch the price for.
+      data_model = contract.key
+
+      # Spread in percent points above which the price is considered stale.
+      spread = contract.value.spread * 2
+
+      # Time in seconds after which the price is considered stale.
+      expiration = contract.value.expiration * 2
```
Comment (deep-quality-dev) [inline config-spectre.hcl:62]: Is this multiplying needed? What's the impact having doubled values?

File: config-spectre.hcl (hunk 1 above)
Comment (teghnet) [reply config-spectre.hcl:42]: Scribe is one contract that has both `IScribe` and `IScribeOptimistic` interfaces.

File: config-spectre.hcl (hunk 2 above)
Comment (teghnet) [reply config-spectre.hcl]: Removed.

**Review by MDobak — APPROVED** [review]

**Pull request merged**